draft-ietf-ppm-dap.md

```diff
@@ -1894,9 +1894,9 @@ following checks:
    invalid with error `invalid_message`.
 
 1. Check if any two public or private report extensions have the same extension
-   type. (For the purposes of this check, a private report extension can
-   conflict with a public report extension.) If so, the Aggregator MUST mark the
-   input share as invalid with error `invalid_message`.
+   type. (For the purposes of this check, a public extension MUST NOT conflict
+   with a private extension.) If so, the Aggregator MUST mark the input share as
```

Comment:
I don't understand the purpose of this parenthetical statement. As jhoyla notes, the old version was ambiguous. The new version is more correct, but now is redundant and I think we should delete it.
```suggestion
   type. If so, the Aggregator MUST mark the input share as
```

@cjpatton do you remember why that sentence in parentheses was ever there?

Reply:
I don't recall discussing this paranthetical in particular.

I think we're attempting to define the following algorithm: let $S$ be the multiset of extension points among the public and private extension fields: if any value occurs twice in $S$, then reject.

I would probably clarify this as follows: "if any extension type occurs twice across the public and private extension fields, then reject".